### DISCUSSION TITLE
Problem with video screen breaking when segments change

### DISCUSSION THREAD

**Discussion by rkselrkswm5** [mifi/lossless-cut#2063] [General]
I am using version 3.61.1.0.

The keyframe cutting method uses ffmpeg -ss -i .... (cut at adjacent keyframes)

When cutting high-resolution VR videos such as HEVC 5760x2880, 7680x3840, 8192x4096, etc., the screen breaks at the segment transition area.

This problem also occurred in previous versions.

**Comment by mifi** [comment]
not sure what you mean by "screen breaks". i hope the glass on your screen doesn't actually break 😜 j/k.

do you mean that when you open the exported output file in VLC, there is visual corruption in the transition points, before recovering again after a few seconds?

have you checked the troubleshooting? https://github.com/mifi/lossless-cut/blob/master/issues.md

**Reply by rkselrkswm5** [reply]
for example
When divided into segments like this
00:01:00 ~ 00:05:00
00:07:00 ~ 00:08:32
00:12:09 ~ 00:15:28

This refers to a phenomenon where certain parts of the video screen are broken in the section that changes from 5 minutes 00 seconds to 5 minutes 01 seconds or from 8:32 seconds to 12 minutes 09 seconds in the combined video.
![20240709_044322](https://github.com/mifi/lossless-cut/assets/153392213/c6f94850-33ac-4894-9b97-dca84ae6e02d)

In this way, the broken screen will return to normal after about 1-2 seconds.

It seems that the key frame was not properly cut when cutting, so the frame was not properly recognized in the screen transition area.

**Reply by rkselrkswm5** [reply]
The method I am currently using as a temporary solution is not to save the video after dividing it into several segments.
 
If I save only one segment and then combine the segments using another program, the problem of the screen breaking as shown in the screenshot above does not occur when the segments are switched.       I tried cutting some videos using this method.
Sometimes there is a section where the screen breaks, and sometimes it doesn't.

This doesn't seem to be a solution.

**Reply by mifi** [reply]
if even disabling all tracks except video doesn't work, and none of the troubleshooting tips help (like keyframe cut, avoid_negative_ts) etc, then I don't know how to fix it unfortunately, it's probably a shortcoming in ffmpeg that gets triggered with your particular file/codec.
